### PR TITLE
USWDS-Site - Changelog: Add entry for high contrast date picker fix [#5701]

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Adds changelog entry for uswds/uswds#5701 and removes the known issue resolved by PR.

## Changelog

>  **Adds focus styles to calendar button in high contrast mode.** Now, the calendar icon changes to the `highlight` high contrast token.

## Removed known issue

> The focus indicator is not visible around the calendar icon in high contrast mode. More information on GitHub: [uswds#5535](https://github.com/uswds/uswds/pull/5535)

## Preview link

[Date picker changelog →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-changelog-5701/components/date-picker/#latest-updates)

### CODE REVIEW

File: _data/changelogs/component-date-picker.yml [hunk 1]
```diff
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Adds focus styles to calendar button in high contrast mode. 
+    summaryAdditional: Now, the calendar icon changes to the `highlight` high contrast token.
```
Comment (amyleadem) [inline _data/changelogs/component-date-picker.yml:7]: Suggesting a couple small changes: Update to past tense and add detail about when the icon changes
```suggestion
    summary: Added focus styles to calendar button in high contrast mode. 
    summaryAdditional: Now, the calendar icon changes to the `highlight` high contrast token on focus.
```

**Comment by mahoneycm** [comment]
@amyleadem committed your suggestion! 👍

**Comment by mahoneycm** [comment]
## Documentation update required

Need to update do remove date picker calendar icon known issuer surrounding focus indicators on the calendar button. This work will be completed in this PR shortly.

**Comment by mahoneycm** [comment]
@amyleadem removed known issue and updated PR description! Ready for re-review

**Review by amyleadem — APPROVED** [review]
LGTM!

**Pull request merged**